site-packages/integralstor/update_system_date_time.py

```diff
@@ -0,0 +1,28 @@
+from integralstor_utils import command
+
```

Comment:
This file should be in integralstor_utils as it is a more general file that can be used by gridcell also. Also, call it something like system_date_time.py